File: messages/src/main/java/com/thundashop/core/ticket/TicketEvent.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.ticket;

import java.util.Date;

/**
 *
 * @author ktonder
 */
public class TicketEvent {
    public TicketEventType eventType;
    public Date date = new Date();
    public String content = "";
    public TicketState state;
    public String updatedByUserId = "";
    public boolean isAnswer = false;
}

File: html5apps/terminal/events/core/ticket/TicketEvent.php
<?php
class core_ticket_TicketEvent {
	/** @var core_ticket_TicketEventType */
	public $eventType;

	/** @var String */
	public $date;

	/** @var String */
	public $content;

	/** @var core_ticket_TicketState */
	public $state;

	/** @var String */
	public $updatedByUserId;

	/** @var String */
	public $isAnswer;

}
?>